src/phoenix/datasets/evaluators.py

```diff
@@ -192,6 +192,75 @@ def _parse_label_from_explanation(raw_string: str) -> str:
     return raw_string
 
 
+class RelevanceEvaluator:
+    annotator_kind = "LLM"
+    template = (
+        "Determine if the following response is relevant to the query. In this context, "
+        "'relevance' means that the response directly addresses the core question or topic of the "
+        "query. First, explain step-by-step why you think the text is or is not relevant. "
+        "Then provide a single word label; 'true' if the text is relevant or 'false' if the text "
+        "is not relevant. "
+        "Here is an example template for your reponse:\n\n"
+        "CRITERIA: the response is 'relevant' to the query\n"
+        "QUERY: *text that contains a query*\n"
+        "RESPONSE: *a response that may or may not be relevant to the query*\n"
+        "EXPLANATION: *a step by step explanation of your reasoning for whether or not the "
+        "response is relevant to the query*\n"
+        "LABEL: *true or false*\n\n"
+        "Follow this template for the following example:\n\n"
+        "CRITERIA: the response is 'relevant' to the query\n"
+        "QUERY: {reference}\n"
+        "RESPONSE: {submission}\n"
+        "EXPLANATION: "
+    )
```

Comment:
Non-blocking: Can we store templates as strings with `"""<template>"""`? This lets us use tools such as [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) to format, otherwise, we have to manually format to avoid line length lint errors when adjusting the template.

Reply:
I think this string format is better. It avoid not-semantically relevant newlines being included that only exist because of our line length limits